core/src/main/java/io/questdb/griffin/engine/functions/catalogue/AttributeCatalogueFunctionFactory.java

```diff
@@ -236,6 +246,12 @@ public int getStrLen(int col) {
         metadata.add(new TableColumnMetadata("attrelid", ColumnType.INT, null));
         metadata.add(new TableColumnMetadata("attname", ColumnType.STRING, null));
         metadata.add(new TableColumnMetadata("attnum", ColumnType.SHORT, null));
+        metadata.add(new TableColumnMetadata("atttypid", ColumnType.INT, null));
```

Comment:
the aim of the SQL is to enum columns. Having constant 0 out of the field isn't going to work